apps/elewa-website/src/app/app.component.html

```diff
@@ -1 +1,10 @@
-<router-outlet></router-outlet>
+<div
+  *ngFor="let content of textContents; let i = index"
+  class="text-content"
+  [ngClass]="{ 'pink-bg': i === 0, 'white-bg': i === 1 }"
+>
+  <h2>{{ content.title }}</h2>
+  <div *ngFor="let description of content.descriptions" class="description">
+    <p class="paragraph">{{ description }}</p>
+  </div>
+</div>
```

Comment:
?? No need for any of this. Also this should have reused the component you created